lib/active_record/connection_adapters/fb/schema_statements.rb

```diff
@@ -144,13 +168,19 @@ def remove_column(table_name, column_name, type = nil, options = {})
         def change_column(table_name, column_name, type, options = {})
           type_sql = type_to_sql(type, *options.values_at(:limit, :precision, :scale))
 
-          execute(squish_sql(<<-end_sql))
+          if type == :text
+            remove_column table_name, column_name
```

Comment:
Change column is supposed to be a "safe" operation, meaning it won't delete your data. I would say dropping a column without warning would be very unsafe.